AirCasting/CoreData/MeasurementStream.swift

```diff
@@ -9,6 +9,8 @@ enum MeasurementStreamSensorName: String {
     case pm1 = "AirBeam3-PM1"
     case pm2_5 = "AirBeam3-PM2.5"
     case pm10 = "AirBeam3-PM10"
+    case mini_pm1 = "AirBeamMini-PM1"
```

Comment:
Airbeam Mini is technically a different sensor so we have to have it separate from the AB3 sensors, even if they are measuring the same thing.